modules/packages/Sort.chpl

```diff
@@ -1733,7 +1733,7 @@ module RadixSortHelp {
   proc msbRadixSortParamLastStartBit(Data:[], comparator) param {
     // Compute end_bit if it's known
     // Default comparator on integers has fixed width
-    const ref element = Data[Data.domain.low];
+    const ref element = Data[Data.domain.lowBound];
```

Comment:
I think this one should be alignedLow based on https://github.com/chapel-lang/chapel/pull/19820#issuecomment-1138770086

Reply:
It appears to me like the radix sort returns `false` (for "I can't handle this") for strided arrays, but I don't see any realistic downside to making this change anyway, so have done so.